nodes/src/nodes/agent_deepagent/deepagent.py

```
@@ -595,6 +625,114 @@ def _log_tool_call(tool_name: str, input: Any = None, output: Any = None) -> Non
+        if not deepagent_node_ids:
+            return []
+
+        from deepagents.middleware.subagents import SubAgent as _SubAgent  # noqa: PLC0415
+
+        subagents: list[Any] = []
+        for node_id in deepagent_node_ids:
+            req = IInvokeDeepagent.Describe()
+            try:
+                self._invoker.instance.invoke('deepagent', req, nodeId=node_id)
+            except Exception:
+                pass
```

Ruff (0.15.9)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(Exception): ...`

(SIM105)